### PR TITLE
update docs/logging.md to v1beta3

### PR DESCRIPTION
**Pull request by caesarxuchao**
update docs/logging.md to v1beta3

@nikhiljindal 

ref #8087

### CODE REVIEW

File: docs/logging.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ As of Kubernetes 0.11, when you create a cluster the console output reports the
 a URL for a [Kibana](http://www.elasticsearch.org/overview/kibana/) dashboard viewer for the logs that have been ingested
 into Elasticsearch.
 ```
-Cluster logs are ingested into Elasticsearch running at https://130.211.152.93/api/v1beta1/proxy/services/elasticsearch-logging/
-Kibana logging dashboard will be available at https://130.211.152.93/api/v1beta1/proxy/services/kibana-logging/ (note the trailing slash)
+Elasticsearch is running at https://104.197.10.10/api/v1beta3/proxy/namespaces/default/services/elasticsearch-logging
```
Comment (nikhiljindal) [inline docs/logging.md]: Dont we need the trailing slash?

@satnam6502

**Comment by nikhiljindal** [comment]
LGTM.
Will wait for @satnam6502 to have a look.

File: docs/logging.md (hunk 1 above)
Comment (caesarxuchao) [inline docs/logging.md]: This is a direct copy of the output of `kubectl cluster-info`. 

I think we need the trailing slash if we want to `curl` the proxy.

Comment (satnam6502) [inline docs/logging.md]: Yes, for Elasticsearch we need the trailing slash so access from curl and other CLI tools.

**Comment by satnam6502** [comment]
I've been working a logging blog article, after which I intend to totally overhaul this page. It's fine to make minor fixes now -- but I suspect they will be short lived.

**Pull request merged**